sky/clouds/service_catalog/data_fetchers/fetch_lambda_cloud.py

```diff
@@ -43,18 +44,19 @@
     'RTX6000': 24576,
     'V100': 16384,
     'H100': 81920,
+    'GENERAL': None
 }
 
 
-def name_to_gpu(name: str) -> str:
+def name_to_gpu_and_cnt(name: str) -> Optional[Tuple[str, int]]:
```

Comment:
could we add a comment for the expected format of the name?

Reply:
Added. Thanks!